.circleci/config.yml

```diff
@@ -51,7 +51,7 @@ workflows:
     jobs:
       - validate-with-maven-script:
           name: "releasenotes-builder"
-          image-name: &custom_img "checkstyle/jdk-11-groovy-git-mvn:11.0.13__3.0.9__2.25.1__3.6.3"
+          image-name: &custom_img "relentlesspursuit/test:11.0.13__3.0.9__2.25.1__3.9.5"
```

Comment:
We need to replace this with new checkstyle dockerhub image when @romani builds and pushes it.

Reply:
Sure thing

Reply:
Am I ready to build and push ?